clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go

```diff
@@ -221,16 +221,26 @@ func (w *waitForControlledPodsRunningMeasurement) gather(c clientset.Interface,
 	w.handlingGroup.Wait()
 	w.lock.Lock()
 	defer w.lock.Unlock()
-	numberRunning := 0
+	var numberRunning, numberTerminated, numberUnknown int
 	for _, checker := range w.checkerMap {
-		if status := checker.getStatus(); status {
+		switch checker.getStatus() {
+		case running:
 			numberRunning++
+		case terminated:
+			numberTerminated++
```

Comment:
This is used only in log.Infof - is that really expected?

Reply:
I treat `numberTerminated` as sanity check. It's only used for logging objects status.